File: RxNetWork-Java/RxNetWorkLibrary/src/main/java/io/reactivex/network/bus/SimpleRxBusCallBack.java
package io.reactivex.network.bus;

/**
 * by y on 2017/5/22
 */

public abstract class SimpleRxBusCallBack<T> implements RxBusCallBack<T> {
    @Override
    public void onBusNext(T t) {

    }

    @Override
    public void onBusError(Throwable throwable) {

    }
}

File: RxNetWorkLibrary/src/main/java/io/reactivex/network/bus/SimpleRxBusCallBack.kt
package io.reactivex.network.bus

/**
 * by y on 2017/5/22
 */

abstract class SimpleRxBusCallBack<T> : RxBusCallBack<T> {
    override fun onBusNext(t: T) {

    }

    override fun onBusError(throwable: Throwable) {

    }
}
